spec/rspec/support/spec/stderr_splitter_spec.rb

```diff
@@ -86,7 +86,9 @@
     splitter.verify_no_warnings!
   end
 
-  unless defined?(RUBY_ENGINE) && RUBY_ENGINE == 'rbx'
+  unless defined?(RUBY_ENGINE) && (RUBY_ENGINE == 'rbx' || RUBY_ENGINE == 'truffleruby')
```

Comment:
Please use this:
https://github.com/rspec/rspec-support/blob/main/lib/rspec/support/ruby_features.rb#L51